components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -259,15 +260,15 @@ function RepositoryNotFoundView(p: { error: StartWorkspaceError }) {
 
       if (!userScopes.includes(missingScope)) {
         setStatusMessage(<div className="mt-2 flex flex-col space-y-8">
-          <p className="text-base text-gray-400 w-96">The repository '{`${repoFullName}`}' may be private. Please authorize Gitpod to access to private repositories.</p>
-          <a className="mx-auto" href={authorizeURL}><button className="secondary">Grant Access</button></a>
+          <p className="text-base text-gray-400 w-96">The repository may be private. Please authorize Gitpod to access to private repositories.</p>
+          <a className="mx-auto" href={authorizeURL}><button>Grant Access</button></a>
```

Comment:
**Praise**: Removing the `secondary` here is such an awesome drive-by fix. 💯 Many thanks @trumbitta and @gtsiolis. ❤️